src/applications/vaos/components/Breadcrumbs.jsx

```diff
@@ -14,11 +14,7 @@ export default function VAOSBreadcrumbs({ children }) {
   const breadcrumbs = useSelector(state => state.appointments.breadcrumbs);
 
   return (
-    <VaBreadcrumbs
-      className="medium-screen:vads-u-padding-x--0 vaos-appts__breadcrumbs"
-      aria-label="Breadcrumbs"
-      role="navigation"
-    >
+    <VaBreadcrumbs aria-label="Breadcrumbs" role="navigation">
```

Comment:
I think you can remove the "aria-label" and "role" from `VaBreadcrumbs` web component since it returns a `nav` element with the aria-label (see screenshot). Also, the HTML `nav` element is automatically communicated as a section having a role of navigation. 

<img width="411" alt="Screen Shot 2022-06-16 at 5 31 51 PM" src="https://user-images.githubusercontent.com/54327023/174199469-decc0d39-028b-4e53-a1ae-097593ed34c3.png">
 of navigation.

```suggestion
    <VaBreadcrumbs>
```

Reply:
Yep, I agree.